//package candy.protocol;
//留言的信息

message LeaveMessage
{
    optional string uid     = 1 ;   //留言者uid
    optional string name    = 2 ;   //留言者的名字
    optional string context = 3 ;   //内容
    optional string time    = 4 ;   //留言时间/分钟
    optional bool   unread  = 5 [default = false] ; //未读状态
    optional string head_url= 6 [default = ""];  //留言者的头像
}